src/main.rs

```diff
@@ -221,6 +221,10 @@ struct Args {
     #[clap(long = "otel-traces-exporter", env = "OTEL_TRACES_EXPORTER")]
     otel_traces_exporter: Option<String>,
 
+    /// Specifies which exporter to use for metrics. Only "otlp" is supported at this time
+    #[clap(long = "otel-metrics-exporter", env = "OTEL_METRICS_EXPORTER")]
+    otel_metrics_exporter: Option<String>,
```

Comment:
Hmm, I think we should use the `hide` clap arg for these two, and add a visible `--otel-exporter` arg that uses the same exporter for all three (metrics, logs, traces), like we have for `--otel-exporter-otlp-endpoint`. I know there's not a [standard env var](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection) corresponding to that, but speaking from experience, I think there's two ways users will want to set OTEL config:
- use the standard env vars (which we shouldn't get in the way of)
- just enable the simplest configuration, which is `otlp` with all 3 data types pointed at the same endpoint

Reply:
So thinking about this some more, I wonder if a shared flag would actually provide any value here, or is it more just for the sake of documentation?

The reason I say that is because outside of OTLP, there currently is no other shared open source format option across all three signals which we would reasonably want to support, so the flag feels a little bit unnecessary.

Reply:
To recap for posterity's sake, we decided not to add this new flag, and we will revisit how these flags are more generally defined in a future PR.